### PR TITLE
added service tests

### PR DESCRIPTION
**Pull request by Mast3rwaf1z**
added a class to test services, so instead of only testing endpoints, we're also testing services

also made removal period configurable

### CODE REVIEW

File: CentralHub.Api.Tests/ServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    public void TestRemoveMeasurement()
+    {
+        _localizationService.AddMeasurements(1, new List<Measurement>(){
+            new Measurement("11:22:33:44:55:66", Measurement.Protocol.Wifi, 1),
+            new Measurement("11:22:33:44:55:66", Measurement.Protocol.Bluetooth, 10)
+        });
+        Assert.That(_localizationService.GetMeasurements(1).Count == 1);
+        _localizationService.AddMeasurements(1, new List<Measurement>(){
+            new Measurement("aa:bb:cc:dd:ee:ff", Measurement.Protocol.Wifi, 1),
+            new Measurement("aa:bb:cc:dd:ee:ff", Measurement.Protocol.Bluetooth, 10)
```
Comment (Taoshix) [inline CentralHub.Api.Tests/ServiceTests.cs:31]: Positive RSSI value? damn, must be a powerful signal you got there.

File: CentralHub.Api.Tests/ServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+class TestLocalizationService : LocalizationService
+{
+    public TestLocalizationService()
+    {
+        MeasurementRemover = new Thread(new ThreadStart(SleepForever));
+    }
+    private void SleepForever()
+    {
+        Thread.Sleep(int.MaxValue);
```
Comment (Taoshix) [inline CentralHub.Api.Tests/ServiceTests.cs:47]: Could -1 also work?

**Review by Taoshix — APPROVED** [review]
Looks alright, only minor things I could point out.

**Review by Casper-NS — APPROVED** [review]
Lgtm

**Review by miniJmo — APPROVED** [review]
I am more a -2 guy myself, but okay I will let it slide this time

**Pull request merged**